portafly/src/i18n/locales/en/accounts.yml

```diff
@@ -1,20 +1,107 @@
+
+team_members_table:
+  aria_label: Team members table
+  aria_label_description: Aria label of the table of team members
+  empty_state: Currently, there are no team members. Add team members. #----PROPOSED
+  empty_state_description: Message to show when there are no team members to show, and prompt the user to add members
+  name_header: Name
+  name_header_description: Column listing names of team members
+  email_header: Email
+  email_header_description: Column listing the email addresses of team members
+  sso_authentication_heaer: SSO authentication
+  sso_authentication_header_description: Column listing if SSO authentication has been configured for team members
+  sso_authentication_value:
+    yes: Yes
+    not_yet: Not yet
+  sso_authentication_value_description: Container of possible values about users having SSO configured
+  role_header: Role
+  role_header_description: Column listing the roles associated to each team member
+  role_header_value: #----PROPOSED
+    admin_you: Admin (you)
+    admin: Admin
+    member: Member
+  role_header_value_description: Container with possible values of the roles of each team member
+  permission_groups_header: Permission groups
+  permission_groups_header_description: Column listing the permisions mapped to each team member
+
+## Tab: Invitations
+## Toolbar
+#-------# Bulk selector: available in shared.yml
+#-------# Delete button: available in shared.yml
+invitations_filter:  #----PROPOSED
+ by_recipient: Recipient
+ by_acceptance: Acceptance
+invitations_filter_description: List containing the criteria to filter invitations
+invitations_typeahead:
+  filter_by_recipient: Filter by recipient
+  filter_by_acceptance: Filter by acceptance
+invitations_typeahead_description: Default text to show when the user filters by recipient or acceptance
+
+invitations_table:
+  aria_label: Invitations table
+  aria_label_description: Aria label of the table of invitations
+  empty_state: Currently, there are no invitations sent. Start sending invitations to your team members. #----PROPOSED
+  empty_state_description: Message to show when there are no invitations to show, and prompt the user to send some invites
+  recipient_header: Recipient
+  recipient_header_description: Column listing the email addresses of invitation recipients
+  sent_header: Sent on UTC time #----PROPOSED
+  sent_header_description: Column listing the date when invitations were sent
+  acceptance_header: Accepted on UTC time #----PROPOSED
+  acceptance_header_description: Column listing the date when the invitations were accepted
+  acceptance_values: #----PROPOSED
+    yes_with_time: Yes - {{utc_time}}
+    not_yet: Not yet
+  acceptance_values_description: Container with the possible responses to whether the invitation has been accepted or not.
+  actions_header: Actions
+  actions_header_description: Column listing the available actions to perform with each invitation
+
```

Comment:
@porueesq UI string is missing for the button link in the contextual menu triggered by the kebab menu:
* Delete